site/soc/software/config/endpoints.yaml

```diff
@@ -605,18 +602,16 @@ data:
         default: null
         public:
           host: identity.DOMAIN
-        admin:
-          host: identity.DOMAIN
       path:
         default: /v3
       scheme:
-        #default: "https"
         default: "http"
+        public: "{{ osh_endpoint_http_scheme }}"
         internal: "http"
       port:
         api:
-          #default: 443
           default: 80
```

Comment:
since you took admin end point off the external domain in the substitution, the goal is to leave admin to be the same as internal. I prefer in that case the admin endpoint to have "identity" and port "5000" instead of "identity" and port "80" which leads to beleive that the admin end point should be exposed on the vip. 

This change forces the user to use openstack client 3.18.0 above which no longer uses admin endpoint for keystone. and we should document that requirement in deployment guide and possibly in ansible playbooks.

Reply:
I am not sure if admin endpoint needs to be exposed from deployer as its primarily used for token validation which is used by services as part of its middleware.
Currently only public endpoints are accessible from deployer. So openstack cli should be using public endpoint only. 
I will change admin endpoint to map to internal endpoint as that's what Guang Yee mentioned about general pattern of cloud 9 service entry.

`dana@ardana-qe006-cp1-c1-m1-mgmt:~> openstack endpoint list --service nova
+----------------------------------+---------+--------------+--------------+---------+-----------+----------------------------------------------+
| ID | Region | Service Name | Service Type | Enabled | Interface | URL |
+----------------------------------+---------+--------------+--------------+---------+-----------+----------------------------------------------+
| 41c157b02fb44842b0a2b2eba786aedc | region1 | nova | compute | True | admin | https://192.168.75.1:8774/v2.1/%(tenant_id)s |
| 8b098a70a0084e3da24ddbe9ce84444a | region1 | nova | compute | True | public | https://10.84.81.4:8774/v2.1/%(tenant_id)s |
| e285ae7e0e83452caaaf6c63563224d0 | region1 | nova | compute | True | internal | https://192.168.75.1:8774/v2.1/%(tenant_id)s |
+----------------------------------+---------+--------------+--------------+---------+-----------+----------------------------------------------+`